### PR TITLE
Prepare release v1.3

### PR DESCRIPTION
**Pull request by bebehei**
As all remaining issues on the `1.3` milestone get closed via their associated PR, I started here to write the changelog and release notes for it.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
 
-## Unreleased
+## 1.3.0 - 2017-12-05
 
 ### Added
-- `ellipsize` option to control how long lines should be ellipsized when `word_wrap` is set to `false`
+- `ellipsize` option to control how long lines should be ellipsized when `word_wrap` is set to `false` (#374)
+- A beginning tilde of a path is now expanded to the home of the current user (#351)
+- The image-path hint is now respected, as GApplications send their icon only via this link (#447)
+- If dunst can't acquire the DBus name, dunst prints the PID of the process, holding the name (#458 #460)
```
Comment (WhyNotHugo) [inline CHANGELOG.md]: The second comma on this line shouldn't be there.

Comment (bebehei) [reply CHANGELOG.md]: Thank you @WhyNotHugo

**Comment by bebehei** [comment]
@tsipinakis I've added now some additonal Information to the Readme. I've previously already created a note in the RADME, but forgot to push this 😂 Here on c3 the world's turning a bit different.

Feel free to commit/amend. I'll maybe won't respond that fast. You can approve yourself. 😉

**Comment by bebehei** [comment]
Waiting for approval ...

waiting

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
 ### Fixed
-- `new_icon` rule being ignored on notifications that had a raw icon
-- Do not replace format strings, which are in notification content
-- DBus related memory leaks closed:
+- `new_icon` rule being ignored on notifications that had a raw icon (#423)
+- Do not replace format strings, which are in notification content (#322 #365)
+- DBus related memory leaks closed: (#397)
   On the DBus connection, all hints never got freed. For raw icons, dunst saved them three times.
+- Dunst crashed on X11 servers with RandR support less than 1.5. (#413 #364)
+- Dunst read silently the default config file, if -conf did not specify a valid file (#452)
```
Comment (thomwiggers) [inline CHANGELOG.md]: should be "silently read" instead of "read silently"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,20 +1,35 @@
...
 ## Changed
-- transient hints are now handled
+- transient hints are now handled (#343 #310)
   An additional rule option (`match_transient` and `set_transient`) is added
-  to optionally reset the transient setting
+  to optionally reset the transient setting 
+- Timeouts are now calculated in microseconds with GLib's `g_get_monotonic_time()` (#379 #291)
+- `icon_folders` is renamed to `icon_path` (#170)
+- `config.def.h` and `config.h` got merged (#371)
+- The dependency to GTK3+ had been removed. Instead of GTK3+, dunst requires now gdk-pixbuf,
```
Comment (thomwiggers) [inline CHANGELOG.md]: dependency on

File: RELEASE_NOTES [hunk 4]
```diff
@@ -1,3 +1,64 @@
...
+	packages for those paths and will put it there.
+
+	To overwrite the pkg-config values, you can manually specify another path.
+	Use `SERVICEDIR_(DBUS|SYSTEMD)` vars as parameters to your make calls.
+
+	For all introduced variables, see [the README.md].
+
+* Portability
+
+  GNU-specific functions had been disabled to make dunst portable to nongnu libc's.
```
Comment (thomwiggers) [inline RELEASE_NOTES]: should probably be "have been"

**Comment by bebehei** [comment]
@thomwiggers Thanks, you're awesome! You made the job, I asked the others for. That's great.

**Comment by tsipinakis** [comment]
Merging this in a temp branch to finalize the release.

**Pull request merged**